tests/Equinox.CosmosStore.Integration/DocumentStoreIntegration.fs

```diff
@@ -495,4 +495,37 @@ type Tests(testOutputHelper) =
         verifyRequestChargesMax 1
     }
 
+    [<AutoData(SkipIfRequestedViaEnvironmentVariable="EQUINOX_INTEGRATION_SKIP_COSMOS")>]
+    let ```Can safely evolve AccessStrategy over time`` cartContext = Async.RunSynchronously <| async {
```

Comment:
I think we have FsCheck3 in da house so it should just work wihtout the RS?

Reply:
It failed locally when I tried

Reply:
I'll never doubt you again, promise!